### PR TITLE
Revive the TilesetBuilder

### LINKED ISSUE #2: Make.cmd
OpenRA has a **make.cmd** file used to download dependencies and compile the source code (from command line). It's very handy for people who haven't installed Visual Studio (which takes a lot of HDD space).

It would be really interesting/useful to have such a file in the TilesetBuilder repository too.

### PR DESCRIPTION
**Pull request by penev92**
This fixes a couple of issues caused by separating it from the main repository, adds a `.sln` to enable MSBUILD to build it directly and adds a build script. Also updates the README with build instructions.

Also closes #2 btw.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,8 @@ Open indexed PNG files and export them into Westwood 2D terrain tile files.
 ![Screenshot](https://camo.githubusercontent.com/0194457c97c9cd10be75cb7be8f62135f4238a55/687474703a2f2f692e696d6775722e636f6d2f33437538392e706e67)
 
 More info on this wiki page: https://github.com/OpenRA/TilesetBuilder/wiki
+
+Build instructions:
+
+Copy the entire TilesetBuilder directory inside OpenRA's root directory. Build OpenRA if you haven't. Open the TilesetBuilder directory and run make.cmd
+ by double-clicking.
```
Comment (Mailaender) [inline README.md]: Spurious newline.

Comment (penev92) [inline README.md]: Fixed.

**Comment by xan2622** [comment]
:+1:

**Comment by Mailaender** [comment]
Thanks!

**Pull request merged**